torch/jit/_script.py

```diff
@@ -787,7 +791,7 @@ def call_prepare_scriptable_func(obj):
     memo: Dict[int, torch.nn.Module] = {}
     return call_prepare_scriptable_func_impl(obj, memo)
 
-def script(obj, optimize=None, _frames_up=0, _rcb=None):
+def script(obj, optimize=None, _frames_up=0, _rcb=None, example_inputs: Optional[List[Tuple]]=None):
```

Comment:
@suo suggested that we should implement new feature in a private API first (something like `_script`) and add additional argument there. Calls to `torch.jit.script` should delegate to `_script` without passing `example_inputs`. This way we can keep the API clean while being able to test out the functionality.